group.go

```diff
@@ -174,6 +176,59 @@ func (cli *Client) UpdateGroupParticipants(jid types.JID, participantChanges map
 	return resp, nil
 }
 
+// GroupRequestParticipantsList get the list of participants waiting to join the specified group in WhatsApp.
+func (cli *Client) GroupRequestParticipantsList(jid types.JID) []*types.JID {
```

Comment:
This should return proper errors.

If you know which types of errors the IQ sending is expected to return, you should add human-readable errors to `errors.go`, then return `wrapIQError(humanReadableErr, err)`

For missing tags, return an `ElementMissingError`